kit/Kit.cpp

```diff
@@ -2830,30 +2851,57 @@ int KitSocketPoll::kitPoll(int timeoutMicroS)
     }
     else
     {
+        // taskPriority: 100==TaskPriority::DEFAULT, 110==TaskPriority::DEFAULT_IDLE
+        const bool loIdleDefault = (taskPriority > 100 && taskPriority <= 110) ||
+                                   (taskPriority == 100 && timeoutMicroS == 0);
+        const bool loIdleLong = taskPriority > 110 ||
+                                (taskPriority == 100 && timeoutMicroS > 2500000); // LO-IdleHandler's SC_IDLE_MAX = 3s, allowing a 500ms lag
```

Comment:
Was there some known problem that prompts the use of SC_IDLE_MAX here? The special casing and then there's a distinction between idle and very idle?

Reply:
The IdleHandler didn't set taskPriority higher (sadly), so I identify it this way. Besides, there is an 'Await' (spellchecker etc) task which also used the magic 3s clearly denoting a long idle period (w/o changing the priority).

Reply:
in the revised version using LibreOfficeKitIdleHintType it could be clarified better. 
However, the SC_IDLE_MAX case on default priority still needs to be detected.